shared-bindings/_bleio/Adapter.c

```diff
@@ -142,6 +142,34 @@ const mp_obj_property_t bleio_adapter_enabled_obj = {
                (mp_obj_t)&mp_const_none_obj },
 };
 
+//|
+//|     tx_power: int
+//|     """transmitter power"""
```

Comment:
What units? Should it be a float?

Reply:
I believe it is an int8_t in the Soft Device API.